java-frontend/src/main/java/org/sonar/plugins/java/api/tree/MethodInvocationTree.java

```diff
@@ -49,6 +49,11 @@ public interface MethodInvocationTree extends ExpressionTree {
 
   Arguments arguments();
 
+  /**
+   * @deprecated in favor of `methodSymbol()`, which returns the narrower type `MethodSymbol` instead of `Symbol`.
+   */
+  @Deprecated(forRemoval = true)
```

Comment:
```suggestion
  /**
   * @deprecated in favor of {@link #methodSymbol()}, which returns the narrower type {@link Symbol.MethodSymbol} instead of {@link Symbol}.
   */
  @Deprecated(since = "7.16", forRemoval = true)
```